github-contribution-guideline.md

```diff
@@ -0,0 +1,122 @@
+# Contribution Guide using Github
+
+This guide describes the process and the conventions for contributing an article using the tools provided by GitHub.
+
+## Prerequisites
+
+1. Get a Github Account if you not yet have one
+   
+2. Make sure you are properly registered as a public member of the "Zühlke Github Community":
+    * Follow these instructions: https://github.com/Zuehlke/core/issues/6
+    * If you can not access this issue: 
+      * send your github username to one of the admins, e.g. Rolf Bruderer
+      * after you received the invitation, accept it
+      * then proceed as described on https://github.com/Zuehlke/core/issues/6
+    
+## Announce
+
+1. **Check for Duplicates:** Is there already an article about your topic?
+  
+   * Please check our [Backlog of Articles](https://github.com/Zuehlke/zuehlke-book/issues?q=is%3Aissue+label%3Aarticle) whether there is already an article existing or planned for similar topic. 
+  
+   * If there is already a similar article, you can contact the author and find a solution (leave the subject to the author, offer your help as reviewer or co-author, take the subject over, or write a second article in response or extension to the first article, ...). 
+   
+2. **Announce a Topic:**
+  
+   * Open a [New Issue for your Article](https://github.com/Zuehlke/zuehlke-book/issues/new?labels=article&title=Article:%20%3Cput%20topic%20title%20here%3E)
+  
+   * Describe roughly the idea you have in mind, what the article should be about.
+  
+   * Assign yourself to the issue.
+
+   * Assign the label "article".
+
+   * If you have a great idea about an article, but would not like to write about yourself? Add your idea to the existing [Ideas for Articles](https://github.com/Zuehlke/zuehlke-book/issues/41) issue!
+
+## Discuss
+
+It proved to be useful to discuss about an articles content before writing the article. Ask your colleagues, maybe they have some valuable input! 
+
+   * Use the [Issues on Github](https://github.com/Zuehlke/zuehlke-book/issues): discuss directly in the issue you created - and don't miss to give feedback to other announced articles.
+
+   * Use Yammer, a coffee break, a team event or similar
+
+## Write 
+
+**Content Conventions:**
+
+* written in English
+
+* short enough to be read in 10 minutes: size should be less than 2 A4 pages long in printout PDF version (that is between 400 and 750 words)
+
+* reflects on a single subject only (better make several articles, if you have too much material / ideas to write about)
+
+* matches our [Vision](./README.md#vision): it is about practicaly proven concepts, best practices, cultural philosophies or great ideas that we use in our daily work at Zühlke.
+
+* the subject is related to our work and the advice concrete enough to consider.
+
+* the subject is something you have experience on in some real projects and not just theoretical ideas
+
+**Language Conventions:**
+
+@Rolf: Please add title guideline
```

Comment:
@bruderol what was that link about the capitalized words in English titles?

Reply:
again?